### ISSUE TITLE
Terraform destroy fails when state is empty

### ISSUE THREAD

**Issue by LeComptoirDesPharmacies** [LeanerCloud/terraform-aws-autospotting#15]
Hi,

When using autospotting module, there is an error if we try to "terraform destroy" while state is empty.
Console log : 
```
Error: Error in function call

  on .terraform\modules\autospotting\AutoSpotting-terraform-aws-autospotting-06901ef\modules\lambda\outputs.tf line 2, in output "function_name":
   2:   value = element(
   3:
   4:
   5:
   6:
   7:
   8:
    |----------------
    | aws_lambda_function.autospotting is empty tuple
    | aws_lambda_function.autospotting_from_s3 is empty tuple

Call to function "element" failed: cannot use element function with an empty
list.
```

This seems to be the same as we can see on this github issue : https://github.com/terraform-aws-modules/terraform-aws-eks/issues/568

Yours faithfully,
LCDP

**Comment by cristim** [comment]
That seems to be a Terraform/AWS provider issue, there's nothing we can do on our side about it, just wait for it to be fixed upstream.

**Comment by cristim** [comment]
Actually looking more carefully at that EKS ticket it turns out there's a workaround.

I have no plans to address this myself, but if it affects you please consider implementing a similar workaround in the AutoSpotting Terraform module and contribute a PR, I'd gladly consider it.

**Comment by cristim** [comment]
@LeComptoirDesPharmacies this should no longer happen with the current version, looks like it was resolved in #10.

I'm closing this issue as fixed but please let me know if you're still seeing this behavior so I reopen it.
